test/data/executor.start.yaml

```diff
@@ -1,3 +1,5 @@
+annotations:
+    beta.screwdriver.cd/executor: k8s
```

Comment:
We have documentation for the executor plugin in the our guide right? I think @tkyi makes a good point about how to document it. Once we have figured out the rules to "Consume annotations to choose executor resource", we can document it.